packages/carbon/README.md

```diff
@@ -0,0 +1,181 @@
+[![Build Status][build-shield]][build-url]
+[![npm][npm-shield]][npm-url]
+[![npm downloads][npm-dl-shield]][npm-dl-url]
+[![Contributors][contributors-shield]][contributors-url]
+[![Apache 2.0 License][license-shield]][license-url]
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/rjsf-team/react-jsonschema-form">
+    <img src="./logo.png" alt="Logo" width="340">
+  </a>
+
+  <h3 align="center">@rjsf/carbon</h3>
+
+  <p align="center">
+  Carbon Design System theme, fields and widgets for <a href="https://github.com/rjsf-team/react-jsonschema-form/"><code>react-jsonschema-form</code></a>.
+    <br />
+    <a href="https://rjsf-team.github.io/react-jsonschema-form/docs/"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://rjsf-team.github.io/react-jsonschema-form/">View Playground</a>
+    ·
+    <a href="https://github.com/rjsf-team/react-jsonschema-form/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rjsf-team/react-jsonschema-form/issues">Request Feature</a>
+  </p>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [About The Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Roadmap](#roadmap)
+- [Contributing](#contributing)
+- [Contact](#contact)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+TODO
+
+[![@rjsf/carbon Screen Shot][product-screenshot]](https://rjsf-team.github.io/@rjsf/carbon)
+
+Exports `carbon` theme, fields and widgets for `react-jsonschema-form`.
+
+### Built With
+
+- [react-jsonschema-form](https://github.com/rjsf-team/react-jsonschema-form/)
+- [Carbon Design System](https://carbondesignsystem.com/)
+- [TypeScript](https://www.typescriptlang.org/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+### Prerequisites
+
+- `@carbon/react >= 1.7.0`
+
+Refer to the [rjsf installation guide](https://rjsf-team.github.io/react-jsonschema-form/docs/#installation) and [Carbon Design System installation guide](https://carbondesignsystem.com/developing/frameworks/react/#install) and for more details.
+
+---
+
+### Installation
+
+```bash
+yarn add @carbon/react
+```
+
+```bash
+yarn add @rjsf/core
+```
+
+<!-- USAGE EXAMPLES -->
+
+## Usage
+
+```js
+import Form from '@rjsf/carbon';
```

Comment:
However you decide to make custom styles work, you might need something like this
```
import Form from '@rjsf/carbon';
import '@rjsf/carbon/styles/styles.css'
```